### DISCUSSION TITLE
[5.x] Disable "Double-click anywhere within a table row" when inline editing is active

### DISCUSSION THREAD

**Discussion by bensomething** [craftcms/cms#15441] [Ideas] [answered]
Being able to double-click anywhere on an element index row to open the side panel is a great addition, but this gets a little janky when inline editing is enabled. The main example of this is when you attempt to double- or triple-click text in a text field to select a word or the whole text. This opens the side panel instead, which isn't expected. I think disabling the double-click action when inline editing is active would fix this.

### Steps to reproduce

1. Open an element index that has a text field showing and enable inline editing.
2. Double click on a word in one of the text fields to select it. The side panel will open instead.

Another issue I've run into in relation to this is when you turn on inline editing, open the side panel for a row, and then save the entry from there. Attempting to then save at the bottom of the index will throw an error.

### Steps to reproduce

1. Open an element index that has a text field showing and enable inline editing.
2. Double click a row to open that entry in the side panel.
3. Edit the text field that is showing in the previous element index and save the entry. That particular field will now be shown as read-only in the element index.
4. Click the **Save** button at the bottom of the element index. This will display a **A server error occurred** message.

I think disabling the ability to open rows when inline editing is enabled would fix that problem as well. Anyway, just my 2¢! Thanks for all your hard work on C5, it's awesome.

**Comment by thupsi** [comment]
Had the same issue recently! +1

**Comment by brandonkelly** [comment, marked as answer]
Thanks for pointing that out! I’ve just disabled element editor slideouts when inline editing is active, for Craft 5.3. (3345511b9c77f6e11184767e1fbbb143dff03bde)
